include/Architecture.hpp

```diff
@@ -364,21 +364,18 @@ class Architecture {
   void createDistanceTable();
   void createFidelityTable();
 
-  static double costHeuristicBidirectional(const Dijkstra::Node& node) {
-    auto length = node.cost - 1;
+  static double dijkstraNodeToCostNonFidelity(const Dijkstra::Node& node) {
```

Comment:
Just a very minor nit: the naming of this function is kind of non-ideal as there is no "fidelity" version. Might as well just call this function `dijkstraNodeToCost`.

Reply:
I am going to submit another PR adding some preparatory functionality for fidelity-aware mapping (including a fidelity-aware version of this method) probably later this day. Since I did not know, if at that point this PR will already be merged, I just wanted to prevent merging conflicts with this naming. But if you want I can also rename it `dijkstraNodeToCost` for now.

Reply:
I thought that this might be the case. I think I still have a slight preference for just removing the "NonFidelity" from the function name as it doesn't really add to the description of what the function does, whereas the fidelity version clearly states that it does something similar to the function here, but also considering fidelity.
However, this is definitely not a strong opinion. So feel free to leave the naming as it is.

Reply:
Oh, I see, that is true. Alright, I changed it now.